datashader/glyphs/trimesh.py

```diff
@@ -214,10 +227,10 @@ def map_onto_pixel(vt, bounds, x, y):
 
         Points falling on upper bound are mapped into previous bin.
```

Comment:
Is this statement still accurate?

Reply:
(Maybe that same point should be made still, but elsewhere, since it's no longer enforced here?)

Reply:
No, it is no longer accurate so I have removed it.